pontoon/contributors/views.py

```diff
@@ -42,7 +42,11 @@ def contributor_email(request, email):
 
 
 def contributor_username(request, username):
-    user = get_object_or_404(User, username=username)
+    try:
+        user = User.objects.get(username=username)
```

Comment:
It would be good to here check if `UserProfile.username` is set, and if so, redirect there.